README.md

```diff
@@ -1,4 +1,33 @@
-# README
 
```

Comment:
@BLaurenB YES! Thank you so much for getting this READme going. Looks great so far!